pages.de/common/pip.md

```diff
@@ -0,0 +1,25 @@
+# pip
+
+> Python package manager.
+> Einige Unterbefehle wie `pip install` sind separat dokumentiert.
+> Weitere Informationen: <https://pip.pypa.io>.
+
+- Installiere ein Paket (siehe `pip install` für weitere Beispiele):
+
+`pip install {{paketname}}`
+
+- Upgrade ein Paket:
+
+`pip install -U {{paketname}}`
+
+- Deinstalliere ein Paket:
+
+`pip uninstall {{paketname}}`
+
+- Speichere eine Liste der installierten Pakete in eine Datei:
+
+`pip freeze > {{requirements.txt}}`
+
+- Zeige die installierten Pakete an:
```

Comment:
Shouldn't that be `Zeige Informattionen über ein installiertes Paket an`?

Reply:
You are right, I'll change it. Thank you